docs/admin/install/venv-redhat.rst

```diff
@@ -9,14 +9,14 @@ Installing on RedHat, Fedora and CentOS
 
    dnf install \
       libxslt-devel libxml2-devel freetype-devel libjpeg-devel zlib-devel libyaml-devel \
-      cairo-devel typelib-1_0-Pango-1_0 gobject-introspection-devel libacl-devel \
+      cairo-devel gobject-introspection-devel libacl-devel \
       python3-pip python3-virtualenv python3-devel git
 
 .. include:: steps/install-system-optional.rst
 
 .. code-block:: sh
 
-    dnf install tesseract-ocr tesseract-devel leptonica-devel
+    dnf install tesseract-devel leptonica-devel
```

Comment:
Language data are needed as well (that's what tesseract-ocr is supposed to pull), so probably this is better:

```suggestion
    dnf install tesseract-langpack-eng tesseract-devel leptonica-devel
```

Reply:
Thanks.